### PR TITLE
PEP8: whitespace in test_stitching.py

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by nicholasreder** [comment]
Great work, peppy compliant

Thanks,

Nick

Nicholas P Reder MD, MPH
Clinical Research Fellow
AP Co-Chief Resident, PGY3
CAP In-Vivo Microscopy Committee, Junior Member

University of Washington Medical Center
Department of Pathology
1959 N.E. Pacific St
Box 356100
Seattle, WA 98195
Pager: 314-8787

File: alpenglow/tests/test_stitching.py [hunk 1]
```diff
@@ -23,27 +23,36 @@
 
 # two channel imaging dataset
 # creates 10 images in z-stack, crisp to blurry with increasing n
-intervals = np.linspace(0.0,3.0,num=10)
-for z in range(1,11):
+intervals = np.linspace(0.0, 3.0, num=10)
```
Comment (arokem) [inline alpenglow/tests/test_stitching.py:26]: This should actually be a function, but let's do this for now

File: alpenglow/tests/test_stitching.py [hunk 2]
```diff
@@ -23,27 +23,36 @@
...
+        strip3_invert = np.pad(strip3_invert, ((0, 0), (19, 19)), 'constant',
+                               constant_values=(0))
         camera_2chan_strip1 = np.vstack((strip1, strip1_invert))
         camera_2chan_strip2 = np.vstack((strip2, strip2_invert))
         camera_2chan_strip3 = np.vstack((strip3, strip3_invert))
         z = int(number*3+1)
-        tiff.imsave("camera_2chan_strip1_z%01d.tif" % z , camera_2chan_strip1)
-        tiff.imsave("camera_2chan_strip2_z%01d.tif" % z , camera_2chan_strip2)
-        tiff.imsave("camera_2chan_strip3_z%01d.tif" % z , camera_2chan_strip3)
+        tiff.imsave("camera_2chan_strip1_z%01d.tif" % z, camera_2chan_strip1)
```
Comment (arokem) [inline alpenglow/tests/test_stitching.py:56]: Maybe the function will return the file names somehow.

**Review by arokem — COMMENTED** [review]
To summarize, this is a first step

**Pull request merged**